transport/x/grpc/integration_test.go

```diff
@@ -159,6 +159,14 @@ func newTestEnv(inboundOptions []InboundOption, outboundOptions []OutboundOption
 	testRouter := newTestRouter(procedures)
 
 	t := NewTransport()
+	if err := t.Start(); err != nil {
+		return nil, err
+	}
+	defer func() {
+		if err != nil {
+			err = multierr.Append(err, t.Stop())
+		}
+	}()
```

Comment:
In this case it is - I only want to call stop if there is an error being returned from the constructor